pkg/function/delete_csi_snapshot_test.go

```diff
@@ -105,7 +105,11 @@ func (testSuite *DeleteCSISnapshotTestSuite) TestDeleteCSISnapshot(c *C) {
 		_, err = fakeCli.CoreV1().PersistentVolumeClaims(testSuite.namespace).Create(ctx, originalPVC, metav1.CreateOptions{})
 		c.Assert(err, IsNil)
 
-		err = fakeSnapshotter.Create(ctx, testSuite.snapName, testSuite.namespace, testSuite.pvcName, &testSuite.volumeSnapshotClass, false, nil, nil)
+		err = fakeSnapshotter.Create(ctx, testSuite.pvcName, &testSuite.volumeSnapshotClass, false,
+			metav1.ObjectMeta{
+				Name:      testSuite.snapName,
+				Namespace: testSuite.namespace,
+			})
```

Comment:
```suggestion
                 snapshotMeta := metav1.ObjectMeta{
				Name:      testSuite.snapName,
				Namespace: testSuite.namespace,
			}
		err = fakeSnapshotter.Create(ctx, testSuite.pvcName, &testSuite.volumeSnapshotClass, false,
			snapshotMeta)
```

Let's do this, like discussed above.